toplevel/byte/topeval.ml

```diff
@@ -87,13 +87,16 @@ let load_lambda ppf lam =
   match
     may_trace := true;
     Fun.protect
-      ~finally:(fun () -> may_trace := false;
-                          if can_free then Meta.release_bytecode bytecode)
+      ~finally:(fun () -> may_trace := false)
```

Comment:
I would be in favor of completely removing the `Fun.protect` call here, if only because it shows up in the user-visible backtrace.

Reply:
Sure, removed.